### PR TITLE
fix: Fix frequent connection errors when making multiple requests sim…

### PR DESCRIPTION
**Pull request by howhelan**
…ultaneously

### CODE REVIEW

**Review by ajpauwels — APPROVED** [review]

**Comment by capt-haddock** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[ajpauwels](https://github.com/pauwels-labs/redact-store/pull/15#pullrequestreview-825103869 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=pauwels-labs%2Fredact-store).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/pauwels-labs/redact-store/blob/main/OWNERS)~~ [ajpauwels]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
